content/docs/ui/sending-email/getting-started-with-automation.md

```diff
@@ -422,9 +422,9 @@ When you re-enable the automation, contacts who previously entered the series wi
 1. Hover over the action menu and select the delete icon.
 1. To delete the automation, click **Confirm**.
 
-## Create a Campaign
+## Create a Single Send
 
-The Automation beta includes the option to create a one-time email campaign, similar to what you find in our current  [Marketing Campaigns]({{root_url}}/ui/sending-email/how-to-send-an-email-with-marketing-campaigns/) product. In the beta, Campaigns does not yet include all functionality that’s available with Marketing Campaigns. Features that are coming soon include: 
+The Automation beta includes the option to create a one-time email campaign, similar to what you find in our current  [Marketing Campaigns]({{root_url}}/ui/sending-email/how-to-send-an-email-with-marketing-campaigns/) product. In the beta, Single Sends do not yet include all functionality that’s available with Marketing Campaigns. Features that are coming soon include: 
```

Comment:
The Automation beta includes the option to create a one-time, one-to-many email campaign, similar...